truffle.js

```diff
@@ -23,6 +24,14 @@ const config = {
       port: 8545,
       network_id: "4"
     },
+    xdai: {
```

Comment:
Just so you know, you can actually create a `truffle-local.js` configuration to supply the overriding HDWallet config with info that won't be checked in nor published.